learningresources/api.py

```diff
@@ -184,11 +191,12 @@ def get_user_tags(repo_id):
 
 def get_resources(repo_id):
     """
-    Get resources from a repository.
+    Get resources from a repository ordered by title.
+
     Args:
         repo_id (int): primary key of the repository
```

Comment:
If we are upcasing the first word like above with the `primary key -> Primary key` changes, we should do that consistently like here and `list of resources`, and `resources` unless I am not grasping the reason for the changes above.